src/uwtools/tests/test_cli.py

```diff
@@ -646,9 +647,10 @@ def test__switch():
 
 
 def test__timedelta_from_str(capsys):
-    assert cli._timedelta_from_str("11:12:13") == dt.timedelta(hours=11, minutes=12, seconds=13)
-    assert cli._timedelta_from_str("11:12") == dt.timedelta(hours=11, minutes=12)
-    assert cli._timedelta_from_str("11") == dt.timedelta(hours=11)
+    assert cli._timedelta_from_str("111:222:333").total_seconds() == 111 * 3600 + 222 * 60 + 333
+    assert cli._timedelta_from_str("111:222").total_seconds() == 111 * 3600 + 222 * 60
+    assert cli._timedelta_from_str("111").total_seconds() == 111 * 3600
+    assert cli._timedelta_from_str("01:15:07").total_seconds() == 1 * 3600 + 15 * 60 + 7
```

Comment:
Adding one example that fits the somewhat expected format with zero-padded entries.

Reply:
Oh, wow -- good call. I got worried for a moment wondering if a leading-zero string might be interpreted as octal or something by `int()` but
```
>>> int("09")
9
```
fortunately not.